code/game/turfs/open/floor/fancy_floor.dm

```diff
@@ -292,3 +292,10 @@
 	underlay_appearance.icon_state = SPACE_ICON_STATE
 	underlay_appearance.plane = PLANE_SPACE
 	return TRUE
+
+/turf/open/floor/carpet/eighties
+	name = "retro floor"
+	desc = "This one takes you back."
+	icon_state = "Stylish 80s arcade carpet, smells like pizza and dropped soda.."
+	floor_tile = /obj/item/stack/tile/eighties
+	broken_states = list("damaged")
```

Comment:
there is no iconstate for damaged in the code, pretty sure this would make it just instantly change to plasteel turfs

Reply:
there wasn't any damage icon on tg either so i assume its intentional